guide/daily-spending-wallet/privacy.md

```diff
@@ -45,222 +46,101 @@ This page should inform about what information is made public when sending or re
+Another solution is to invoke the wallet’s hidden state as a default when the app is opened, to protect against prying eyes during initial display. The pre-hidden state can be unveiled after a tap, PIN entry, or perhaps a short 5-second timer.
 
-An advantage here is the risk of an unauthorized person revealing their information is minimal due to the fact that the toggle isn't immediately visible on the home screen. The downside is that a user cannot quickly hide their information if the need arises.
+This gives users some time to assess their environment before their info is displayed but could leave them frustrated, having to wait for their information to be revealed, especially in an urgent situation.
 
 </div>
 
----
-
-### Design with privacy in mind
-
-Thinking about privacy is critical during the design process. Your users will not have the same level of knowledge of how to use bitcoin privately. The [privacy by design framework](https://www.ipc.on.ca/wp-content/uploads/Resources/7foundationalprinciples.pdf) states that privacy should be incorporated and built into products by default. This way, whether or not the user is concerned with their data privacy, they would always be protected through good UX and UI.
+## Other privacy aspects
 
-However, it still is important to help users understand any actions that might impact their privacy. Most of the risks occur at the point of creating a transaction or requesting a payment, and we should try to design solutions that reduce the risk of unknowingly degrading privacy.
-
-While there is no perfect solution that will guarantee 100% privacy, try to minimize how much information gets shared to the most essential. Consider ways to inform and prevent user actions that negatively impact their privacy as they use your product.
-
-<!--
-There is no perfect solution to guarantee 100% privacy that lasts forever because things can be revealed over time. Since transactions are forever public, even if all precautions are taken at the time of payment to ensure the highest degree of anonymity, future behaviors of the wallet owner or transacting parties can still degrade previously attained privacy. A high amount of diligence is necessary whenever users are transacting with bitcoin. The product should be able to guide, inform, and prevent them against privacy degrading actions.
--->
+On-chain [address re-use]({{ "/guide/how-it-works/wallet-privacy/#generate-a-new-address-for-each-on-chain-payment" | relative_url }}) is a common privacy problem due to the tracability across transactions. Addresses this wallet uses are swap addresses generated by a lightning service provider. This lifts the responsibility for address management out of the hands of the user. More on the [requesting page]({{ "/guide/daily-spending-wallet/requesting/" | relative_url }}).
```

Comment:
Can remove, this wallet only uses swap addresses which are controlled by the LWS.

Reply:
That's what this paragraph explains.